opencodelists/management/commands/import_data.py

```diff
@@ -14,23 +19,94 @@ def iter_possible_modules():
         yield head.replace(os.sep, ".")
 
 
+def valid_from(input_date):
+    try:
+        return datetime.strptime(input_date, "%Y%m%d")
+    except ValueError:
+        raise argparse.ArgumentTypeError(f"Not a valid date (YYYYMMDD): {input_date}")
+
+
 class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument("dataset")
         parser.add_argument("release_dir")
+        parser.add_argument("--release", dest="release_name", help="Release name")
+        parser.add_argument(
+            "--valid-from",
+            type=valid_from,
+            help="For coding system imports: date the release is valid from, in YYYYMMDD format",
+        )
+        parser.add_argument(
+            "--import-ref",
+            help="For coding system imports: optional reference for this import",
+        )
+        parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Force an overwrite of a coding system release",
+        )
 
-    def handle(self, dataset, release_dir, **kwargs):
+    def handle(
+        self,
+        dataset,
+        release_dir,
+        release_name,
+        valid_from,
+        import_ref,
+        force,
+        **kwargs,
+    ):
         try:
             mod = import_module(dataset + ".import_data")
         except ModuleNotFoundError:
-            print(f"Could not find an 'import_data.py' module at path '{dataset}'")
-            print("Maybe you meant one of these module paths?")
+            self.stdout.write(
+                f"Could not find an 'import_data.py' module at path '{dataset}'"
+            )
+            self.stdout.write("Maybe you meant one of these module paths?")
 
             possibilities = list(iter_possible_modules())
             for possibility in possibilities:
-                print(possibility)
+                self.stdout.write(possibility)
 
             sys.exit(1)
 
+        module_name = dataset.rsplit(".", 1)[-1]
+        if module_name in CODING_SYSTEMS:
+            import_data_kwargs = self.get_coding_system_kwargs(
+                module_name, release_name, valid_from, import_ref, force
+            )
+        else:
+            import_data_kwargs = {}
+
         fn = getattr(mod, "import_data")
-        fn(release_dir)
+        fn(release_dir, **import_data_kwargs)
+
+    def get_coding_system_kwargs(
+        self, coding_system, release_name, valid_from, import_ref, force
+    ):
+        if not all([release_name, valid_from]):
```

Comment:
This is an example of some code that would be simplified if we split out the two commands, because we could make `release_name` and `valid_from` required arguments with argparse.